src/partials/layout.html

```diff
@@ -56,7 +56,7 @@
           </a>
         </div>
         <div class="flex-grow flex items-center justify-center">
-          <nav class="flex space-x-12 text-gray-600">
+          <nav id="menu" class="flex space-x-12 text-gray-600">
```

Comment:
`id`s should be unique, so we need a different selector :)

```suggestion
          <nav data-el-menu class="flex space-x-12 text-gray-600">
```